examples/portal_advanced_security.py

```diff
@@ -0,0 +1,34 @@
+# pip install pyjwt
+import jwt
+
+from flask import Flask, request, abort
+
+app = Flask(__name__)
+
+# Somewhere in the config module
+app.config['DEPLOYMENT_1_MASTER_KEY'] = '123'
+app.config['DEPLOYMENT_1_ID'] = 'deployment_1'
```

Comment:
Also here, `deployment-1-pk`, instead of `deployment_1`

Reply:
deployment-1-pk is fine as long as you explain it in your docs

Reply:
Well, we can do the same as for master key, just provide a sample deployment id with `sample` at the end: `deployment_cQPneKKawjVsample`
Is it ok?